packages/cli/src/tests/utils/build.js

```diff
@@ -55,7 +56,7 @@ describe('build (runs slowly)', function () {
     smartPaths.should.not.containEql('node_modules/jest/package.json');
     smartPaths.should.not.containEql('node_modules/node-fetch/README.md');
 
-    smartPaths.length.should.be.within(200, 306);
+    smartPaths.length.should.be.within(200, 324);
```

Comment:
I compared the before/after here ( [gist](https://gist.github.com/Natay/0307723c28f54bb2c5651e71b3d3a667) ):

These come from additional transitive dependencies that are legitimate and expected. Specifically main culprits are:
- form-data: `4.0.1 → 4.0.4`
- semver: 7.7.1 → 7.7.2 (in both packages)
- Various @oclif/* packages

A majority are coming from [oclif](https://github.com/zapier/zapier-platform/pull/1079/files) packages and [ form-data ](https://github.com/zapier/zapier-platform/pull/1079/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR6479)transitive dependencies and